Assets/scripts/Levels/SceneLoader.cs

```diff
@@ -24,6 +35,7 @@ public void LevelSelectionPopup()
     
     public void LoadLobbyScene()
     {
+        Time.timeScale = 1f;
```

Comment:
The purpose of timescale is for pausing and resuming the game it should not be done at the time of level completion.